src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/Commons.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.plugin.dsl
+
+/**
+ * TODO
```

Comment:
I think we can publish the names of the basic extensions so that users can make calls in the user code like
```
extensions.findByName(PROJECT_EXTENSION_NAME)
```